src/components/BookCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { VscHeart } from "react-icons/vsc";
+import ReactStars from "react-rating-stars-component";
+
+function BookCard() {
```

Comment:
This should be a component the receives data as props ex: BookCard({image, title, genera, rating,}). 
Then you use those props in their correct places.

Reply:
the changes have been made